Secrets generating commands not compatible with openssh 7.9

Hi,

we are following the README section to generate secrets (https://github.com/concourse/concourse-chart#secrets). Using openssh 7.9
the documented commands (e.g. `ssh-keygen -t rsa -f host-key  -N ''`) generate keys that don't work with the helm chart. The web pod dies with the following error:

```
kubectl logs -f -n concourse concourse-web-776f984dfc-gv2wr                                                                                                                                   
{"timestamp":"2020-01-14T11:37:52.542320328Z","level":"info","source":"atc","message":"atc.cmd.start","data":{"session":"1"}}                                                                 
{"timestamp":"2020-01-14T11:37:52.617416433Z","level":"info","source":"atc","message":"atc.credential-manager.configured credentials manager","data":{"name":"kubernetes","session":"6"}}     
{"timestamp":"2020-01-14T11:37:52.641506010Z","level":"info","source":"atc","message":"atc.cmd.finish","data":{"duration":93901,"session":"1"}}                                               
panic: runtime error: invalid memory address or nil pointer dereference                                                                                                                       
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1100805]                                                                                                                       
                                                                                                                                                                                              
goroutine 1 [running]:                                                                                                                                                                        
crypto/rsa.(*PrivateKey).Public(0x0, 0x0, 0x0)                                                                                                                                                
        /usr/local/go/src/crypto/rsa/rsa.go:100 +0x5                                                                                                                                          
golang.org/x/crypto/ssh.NewSignerFromSigner(0x7f2db321ba48, 0xc000010010, 0xc000010010, 0x7f2db321ba48, 0xc000010010, 0xeb0c01)                                                               
        /tmp/build/1c3187db/gopath/pkg/mod/golang.org/x/crypto@v0.0.0-20191119213627-4f8c1d86b1ba/ssh/keys.go:720 +0x35                                                                       
golang.org/x/crypto/ssh.NewSignerFromKey(0x2b7f860, 0xc000010010, 0x7f2db32506d0, 0x0, 0xef2fc8, 0xc00118d198)                                                                                
        /tmp/build/1c3187db/gopath/pkg/mod/golang.org/x/crypto@v0.0.0-20191119213627-4f8c1d86b1ba/ssh/keys.go:695 +0x167                                                                      
github.com/concourse/concourse/tsa/tsacmd.(*TSACommand).configureSSHServer(0xc0001ac540, 0xc00005ac00, 0xc000d54ea0, 0x1, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0, ...)                                  
        /tmp/build/1c3187db/concourse/tsa/tsacmd/command.go:186 +0x2a1                                                                                                                        
github.com/concourse/concourse/tsa/tsacmd.(*TSACommand).Runner(0xc0001ac540, 0xc000d54970, 0x0, 0x1, 0x36a28e0, 0xc000dc7080, 0x0, 0x0)                                                       
        /tmp/build/1c3187db/concourse/tsa/tsacmd/command.go:97 +0x27a                                                                                                                         
main.(*WebCommand).Runner(0xc000690d88, 0xc000d54970, 0x0, 0x1, 0x7, 0x6, 0xc000d547e0, 0xc00027d0c0)                                                                                         
        /tmp/build/1c3187db/concourse/cmd/concourse/web.go:56 +0xf4 
```

After reading these docs: https://concourse-ci.org/concourse-generate-key.html we realized that `concourse` binary can be used to generate the same secrets. Using `concourse` to generate the secrets make it work. We used commmands like:

```
docker run -v $PWD:/keys --rm -it concourse/concourse generate-key -t rsa -f /keys/session-signing-key
docker run -v $PWD:/keys --rm -it concourse/concourse generate-key -t ssh -f /keys/worker-key
docker run -v $PWD:/keys --rm -it concourse/concourse generate-key -t ssh -f /keys/host-key
```

It seems like `concourse` binary implements the ssh-keygen command internally but is probably not compatible with the latest openssh (?).

In any case, the README should be aligned with the concourse-ci.org documentation so it might be better if the `ssh-keygen` command are changed with other's using the `concourse` binary (even the working ones listed above).

--- a/README.md
+++ b/README.md
@@ -272,17 +272,31 @@ Be careful with introducing trailing newline characters; following the steps bel
 #
 mkdir concourse-secrets
 cd concourse-secrets
+```
 
-# Generate the files for the secrets that are required:
-# - web key pair,
-# - worker key pair, and
-# - the session signing token.
-#
-ssh-keygen -t rsa -f host-key  -N ''
+Concourse needs three sets of key-pairs in order to work:
+- web key pair,
+- worker key pair, and
+- the session signing token.
+
+You can generate all three key-pairs by following either of these two methods:
+
+##### Concourse Binary
+
+```sh
+docker run -v $PWD:/keys --rm -it concourse/concourse generate-key -t rsa -f /keys/session-signing-key
+docker run -v $PWD:/keys --rm -it concourse/concourse generate-key -t ssh -f /keys/worker-key
+docker run -v $PWD:/keys --rm -it concourse/concourse generate-key -t ssh -f /keys/host-key
+```
+
+##### ssh-keygen
+
+```sh
+ssh-keygen -t rsa -f host-key  -N '' -m PEM
 mv host-key.pub host-key-pub
-ssh-keygen -t rsa -f worker-key  -N ''
+ssh-keygen -t rsa -f worker-key  -N '' -m PEM
 mv worker-key.pub worker-key-pub
-ssh-keygen -t rsa -f session-signing-key  -N ''
+ssh-keygen -t rsa -f session-signing-key  -N '' -m PEM
 rm session-signing-key.pub
 printf "%s:%s" "concourse" "$(openssl rand -base64 24)" > local-users
 ```
@@ -290,8 +304,6 @@ printf "%s:%s" "concourse" "$(openssl rand -base64 24)" > local-users
 All the worker-specific secrets, namely, `workerKey`, `workerKeyPub`, `hostKeyPub` are to be added to a separate Kubernetes secrets object with the name [release name]-worker.
 
 All other secrets are to be added to a secrets object with the name `[release name]-web`.
-
-For the time being, the secret `workerKeyPub` is to be added to both the worker and the web secret objects, until investigated within issue #13019.
 
 You'll also need to create/copy secret values for optional features. See [templates/web-secrets.yaml](templates/web-secrets.yaml) and [templates/web-secrets.yaml](templates/web-secrets.yaml)  for possible values.
 

